webhook-service/lib/denylist_provider.go

```diff
@@ -0,0 +1,77 @@
+package lib
+
+import (
+	"context"
+	"strings"
+
+	keptnkubeutils "github.com/keptn/kubernetes-utils/pkg"
+	logger "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type IDenyListProvider interface {
+	GetDenyList() []string
```

Comment:
would just `Get []string` be enough to make it less verbose (`denyListProvider.getDenyList()` vs `denyListProvider.Get()`